geode-redis/src/test/java/org/apache/geode/redis/internal/executor/key/ExpireAtExecutorJUnitTest.java

```diff
@@ -34,30 +37,30 @@
 
   @Test
   public void calledWithTooFewCommandArguments_returnsError() {
-    List<byte[]> commandsAsBytesWithTooFewArguments = new ArrayList<>();
-    commandsAsBytesWithTooFewArguments.add("EXPIREAT".getBytes());
-    commandsAsBytesWithTooFewArguments.add("key".getBytes());
-    Command command = new Command(commandsAsBytesWithTooFewArguments);
+    List<byte[]> commandAsBytes = new ArrayList<>();
+    commandAsBytes.add("EXPIREAT".getBytes());
+    commandAsBytes.add("key".getBytes());
+    Command command = new Command(commandAsBytes);
 
-    RedisResponse response =
-        new ExpireAtExecutor().executeCommand(command, mockContext());
+    Throwable thrown = catchThrowable(() -> command.execute(mockContext()));
 
-    assertThat(response.toString()).startsWith("-ERR The wrong number of arguments");
+    AssertionsForClassTypes.assertThat(thrown).hasMessageContaining("wrong number of arguments");
+    AssertionsForClassTypes.assertThat(thrown).isInstanceOf(RedisParametersMismatchException.class);
   }
 
   @Test
   public void calledWithTooManyCommandArguments_returnsError() {
-    List<byte[]> commandsAsBytesWithTooFewArguments = new ArrayList<>();
-    commandsAsBytesWithTooFewArguments.add("EXPIREAT".getBytes());
-    commandsAsBytesWithTooFewArguments.add("key".getBytes());
-    commandsAsBytesWithTooFewArguments.add("1".getBytes());
-    commandsAsBytesWithTooFewArguments.add("extra-argument".getBytes());
-    Command command = new Command(commandsAsBytesWithTooFewArguments);
+    List<byte[]> commandsAsBytes = new ArrayList<>();
+    commandsAsBytes.add("EXPIREAT".getBytes());
+    commandsAsBytes.add("key".getBytes());
+    commandsAsBytes.add("1".getBytes());
+    commandsAsBytes.add("extra-argument".getBytes());
+    Command command = new Command(commandsAsBytes);
 
-    RedisResponse response =
-        new ExpireAtExecutor().executeCommand(command, mockContext());
+    Throwable thrown = catchThrowable(() -> command.execute(mockContext()));
 
-    assertThat(response.toString()).startsWith("-ERR The wrong number of arguments");
+    AssertionsForClassTypes.assertThat(thrown).hasMessageContaining("wrong number of arguments");
+    AssertionsForClassTypes.assertThat(thrown).isInstanceOf(RedisParametersMismatchException.class);
```

Comment:
Maybe the same thing, but at least for consistency, `assertThat` should come from the `org.assertj.core.api.Assertions` package.

Reply:
Definitely. Editing those now.